### PR TITLE
Fix edges_len returning wrong number

### PR DESCRIPTION
**Pull request by ricopinazo**

### CODE REVIEW

File: docbrown/core/src/graph.rs [hunk 1]
```diff
@@ -170,7 +170,7 @@ impl TemporalGraph {
     }
 
     pub fn edges_len(&self) -> usize {
```
Comment (ricopinazo) [inline docbrown/core/src/graph.rs:172]: Is this function intended to be exposed to the user? I would change the name to out_edles_len, but if it is exposed to the user I would create another one instead

**Review by fabianmurariu — APPROVED** [review]
LGTM (unless you want to also add `in_edges_len`

File: docbrown/core/src/graph.rs (hunk 1 above)
Comment (fabianmurariu) [reply docbrown/core/src/graph.rs:172]: TemporalGraph should not be exposed to the user, it's GraphDB that should be

Comment (ricopinazo) [reply docbrown/core/src/graph.rs:172]: Ok, in that case I'm going to rename edges_len to out_edges_len in the internal classes to avoid confusion

**Review by fabianmurariu — APPROVED** [review]
LGTM

**Pull request merged**